frontend/src/features/public-form/utils/inputTransformation.ts

```diff
@@ -62,41 +66,55 @@ const transformToVerifiableOutput = <
   input: unknown,
 ): VerifiableAnswerOutput<F> => {
   if (
-    !validateVerifiableInput(input) ||
-    (schema.isVerifiable && !input.signature)
+    input !== undefined &&
```

Comment:
why do you need this here when `validateVerifiableInput` already has a check for `typeof input !== 'object'`?